alonzo/impl/src/Cardano/Ledger/Alonzo.hs

```diff
@@ -214,6 +199,7 @@ instance CC.Crypto c => EraModule.SupportsSegWit (AlonzoEra c) where
   fromTxSeq = Alonzo.txSeqTxns
   toTxSeq = Alonzo.TxSeq
   hashTxSeq = Alonzo.hashTxSeq
+  unsafeApplyTx (Tx bod wit auxdata) = ValidatedTx bod wit (IsValidating True) auxdata
```

Comment:
This method should not exist in this class

Reply:
Moved to another class. Good idea.